mlflow/transformers/__init__.py

```diff
@@ -475,15 +497,42 @@ def save_model(
 
     code_dir_subpath = _validate_and_copy_code_paths(code_paths, str(path))
 
-    _validate_transformers_model_dict(transformers_model)
-
     if isinstance(transformers_model, transformers.Pipeline):
+        _validate_transformers_model_dict(transformers_model)
         built_pipeline = transformers_model
     elif isinstance(transformers_model, dict):
+        _validate_transformers_model_dict(transformers_model)
         built_pipeline = _build_pipeline_from_model_input(transformers_model, task=task)
+    elif isinstance(transformers_model, str):
+        # When a string is passed, it should be a path to model checkpoint in local storage or DBFS
+        if transformers_model.startswith("dbfs:"):
+            # Replace the DBFS URI to the actual mount point
+            transformers_model = transformers_model.replace("dbfs:", "/dbfs", 1)
+
+        if task is None:
```

Comment:
Let's make sure to clearly spell out this behavior in the docs and warn heavily about what happens if the task type is wrong (they will only find out when loading the model)

Reply:
True, this can be error prone. I will make sure including that in the doc update PR.